### ISSUE TITLE
bytes in dict don't works correctly

### ISSUE THREAD

**Issue by EkremDincel** [chr15m/PodSixNet#34]
I am running ``rencode.py`` and I am getting those output in shell:

````python
>>> d = {b"a":1}
>>> loads(dumps(d))
{'a': 1}
>>> loads(dumps(d)) == d
False
````
Same for bytes as values. I am using python3.8.0

**Comment by chr15m** [comment]
@EkremDincel yes the encoder is unable to distinguish between strings and bytes. At some point it would be good to replace it with something which can encode/decode arbitrary Python structures.

**Comment by EkremDincel** [comment]
Last version of [rencode](https://pypi.org/project/rencode/) works well with bytes.

**Comment by chr15m** [comment]
Great. If somebody submits a PR implementing this and the tests pass, I'll merge it.

**Comment by pauliyobo** [comment]
@chr15m  The last rencode version has been made with cython. This means that unlike the version we have now, we can't include the pyx file and utilize it directly.
We should either include the built extension in the package if we want to keep rencode as part of the PodSixNet package, or we should keep it externally.
What's the approach that you'd prefer?

**Comment by chr15m** [comment]
@pauliyobo maybe it makes sense to put it in requirements.txt and have it as an external dependency now. I guess that's pretty standard.
